apps/fulfillment/mock/db.json

```diff
@@ -130,21 +130,9 @@
                 "type": "picking-list-items",
                 "id": "80839794-a5dc-58e7-b95f-55667543258f"
               },
-              {
-                "type": "picking-list-items",
-                "id": "c3d7aa66-72bb-5d29-8a35-8d440910f4f6"
-              },
-              {
-                "type": "picking-list-items",
-                "id": "20415014-987d-528d-b22e-35aeba72f03a"
-              },
```

Comment:
Why did you change mock data?

Reply:
This picking list, had repeated products
![Screenshot 2023-04-06 at 16 41 13](https://user-images.githubusercontent.com/105730125/230429564-39168932-e811-45ff-8ed4-d63362726976.png)